recipes/configs/phi3/mini_full.yaml

```diff
@@ -45,30 +39,34 @@ checkpointer:
   model_type: PHI3_MINI
 resume_from_checkpoint: False
 
+# Dataset
+dataset:
+  _component_: torchtune.datasets.alpaca_cleaned_dataset
```

Comment:
Should we default to a chat dataset since our base models are already instruct tuned?